ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/cio/NettyResponsePipeline.kt

```diff
@@ -7,6 +7,8 @@ package io.ktor.server.netty.cio
 import io.ktor.util.cio.*
 import io.ktor.http.*
 import io.ktor.server.netty.*
+import io.ktor.server.netty.http2.*
+import io.ktor.server.netty.http2.NettyHttp2ApplicationResponse
```

Comment:
Why do we need ` import io.ktor.server.netty.http2.NettyHttp2ApplicationResponse`?